segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessorTest.java

```diff
@@ -239,7 +239,7 @@ public void testReadSegmentWithCancellationException() {
     @Test(timeout = 20000)
     public void testReadSegmentTruncated() {
         // Set up PravegaRequestProcessor instance to execute read segment request against
-        String streamSegmentName = "testReadSegment";
+        String streamSegmentName = "scope/stream/testReadSegment";
```

Comment:
Why did you change all of these names?

Reply:
Segment name must be qualified containing at least stream.
The old unit test code works with unqualified name because the old code doesn't have to retrieve stream/segment information from the name.